tests/Fixer/Alias/MbStrFunctionsFixerTest.php

```diff
@@ -79,5 +79,9 @@ public function strtolower($a);
             '<?php $a = mb_str_split($a);',
             '<?php $a = str_split($a);',
         ];
+
+        if (\PHP_VERSION_ID >= 8_03_00) {
```

Comment:
side note as data providers purist.

data providers should not provide "sometimes 10, sometimes 11" test cases, depends on env.

I do not want to see "you run 10/10" and thinking all tests are executed, I shall see "you run 10/11 and 1 skipped".

Reply:
Done in #7505